CHANGELOG.md

```diff
@@ -2,10 +2,55 @@
 
 ## Table of Contents
 
+- [r0.2.0](#r020)
 - [r0.2.0-rc.2](#r020-rc2)
 - [v0.2.0-rc.1](#v020-rc1)
 - [v0.1.0 - Initial version](#v010---initial-version)
 
+# r0.2.0
+
+**This is the public release of "Identity And Consent Management" version 0.2.0**
+
+## Please note:
+
+* The content of the release includes the "Identity And Consent Management" approved deliverables in **[documentation](https://github.com/camaraproject/IdentityAndConsentManagement/tree/r0.2.0/documentation)** folder.
+* The document [Authentication and Authorization Concept for Service APIs](https://github.com/camaraproject/IdentityAndConsentManagement/blob/release-0.1.0/documentation/CAMARA-AuthN-AuthZ-Concept.md) was part of the 0.1.0 release. **It is deprecated** and it has been removed after this public release of "Identity and Consent Management" version 0.2.0.
```

Comment:
Slight change, as the document is at the time of the merge of this PR no  longer there and it hasn't been removed *after* the release:

```suggestion
* The document [Authentication and Authorization Concept for Service APIs](https://github.com/camaraproject/IdentityAndConsentManagement/blob/release-0.1.0/documentation/CAMARA-AuthN-AuthZ-Concept.md) was part of the 0.1.0 release. **It has been deprecated** and been removed in the course of the public release of "Identity and Consent Management" version 0.2.0.
```